drivers/disp_dev/disp_dev.c

```diff
@@ -21,9 +21,47 @@
 #include <assert.h>
 #include <stdbool.h>
 #include <inttypes.h>
+#include <errno.h>
 
 #include "disp_dev.h"
 
+
+disp_dev_reg_t *disp_dev_reg = NULL;
+
+int disp_dev_reg_add(disp_dev_reg_t *dev)
```

Comment:
Any reason for *not* reusing `clist.h` here?

Reply:
Yes the code is inspired from the saul registry mechanism which doesn't make use of it. Do you think it's worth ?

Reply:
Only if it reduces the amount of code in this PR :)

Reply:
Can we leave this as a follow-up ? Otherwise, I fear this will never get in (already opened 9 months ago).